Tuist/ProjectDescriptionHelpers/Scheme+Initializing.swift

```diff
@@ -0,0 +1,36 @@
+import ProjectDescription
+
+extension Scheme {
+
+    public static func mainScheme(name: String) -> Scheme {
+        return Scheme(
+            name: name,
+            shared: true,
+            buildAction: BuildAction(targets: ["\(name)"]),
+            testAction: TestAction(
+                targets: ["\(name)Tests", "\(name)UITests"],
+                configurationName: "Debug"
+            ),
+            runAction: RunAction(configurationName: "Debug"),
+            archiveAction: ArchiveAction(configurationName: "Release"),
+            profileAction: ProfileAction(configurationName: "Debug"),
+            analyzeAction: AnalyzeAction(configurationName: "Debug")
```

Comment:
Please add `#warning` or comments for these configuration names, we should use `Debug Production` and `Release Production`.